### ISSUE TITLE
Add support for rollup@3

### ISSUE THREAD

**Issue by tbouffard** [tivac/rollup-plugin-sizes#142]
Hi and thanks for this plugin

rollup-plugin-sizes@1.0.4 declares `"rollup": "^2.0.0"` in peerDependencies

Running npm install in a projet using rollup@3 generates the following error
```
npm ERR! While resolving: rollup-plugin-sizes@1.0.4
npm ERR! Found: rollup@3.7.5
npm ERR! node_modules/rollup
npm ERR!   dev rollup@"~3.7.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^2.0.0" from rollup-plugin-sizes@1.0.4
npm ERR! node_modules/rollup-plugin-sizes
npm ERR!   dev rollup-plugin-sizes@"~1.0.4" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: rollup@2.79.1
npm ERR! node_modules/rollup
npm ERR!   peer rollup@"^2.0.0" from rollup-plugin-sizes@1.0.4
npm ERR!   node_modules/rollup-plugin-sizes
npm ERR!     dev rollup-plugin-sizes@"~1.0.4" from the root project
```

Workaround: use the `--legacy-peer-deps` option

**Comment by tivac** [comment]
Fixed in `rollup-plugin-sizes@1.0.5`

https://github.com/tivac/rollup-plugin-sizes/releases/tag/v1.0.5

**Comment by tbouffard** [comment]
Thanks for the quick implementation 🎉
